### PR TITLE
Add partner ref. app to sample projects

### PR DESCRIPTION
**Pull request by chgeo**

### CODE REVIEW

**Comment by chgeo** [comment]
@mrl-sap @ch-sap hope this is fine for you to appear on https://cap.cloud.sap/docs/get-started/samples
Copied the text from your repo's readme.  You might want to tweak it.

File: get-started/samples.md [hunk 1]
```diff
@@ -98,3 +98,16 @@ The Sustainable SaaS (SusaaS) sample application has been built in a partner col
...
 [![Node.js logo](../assets/logos/nodejs.svg 'Link to the Sustainable SaaS (SusaaS) repository.'){style="height:2.5em; display:inline; margin:0 0.2em;"}](https://github.com/SAP-samples/btp-cap-multitenant-saas)
+
+
+## Partner Reference App {.github}
+
+With this application, we give SAP partners guidance on how to extend SAP ERP solutions with side-by-side cloud applications running on SAP Business Technology Platform (SAP BTP). It includes:
+
+- a preselection of SAP BTP components with architecture guidance tailored to meet the cost requirements of small and midsize enterprises (SME),
+- best practices (golden path) to build, deploy, and provision full-stack SAP BTP applications, and
+- information on the interoperability and integration with cloud ERP solutions such as SAP Business ByDesign and SAP S/4HANA Cloud Public Edition.
```
Comment (renejeglinsky) [inline get-started/samples.md:109]:
```suggestion
- A preselection of SAP BTP components with architecture guidance tailored to meet the cost requirements of small and midsize enterprises (SME).
- Best practices (golden path) to build, deploy, and provision full-stack SAP BTP applications.
- Information on the interoperability and integration with cloud ERP solutions such as SAP Business ByDesign and SAP S/4HANA Cloud Public Edition.
```

File: get-started/samples.md [hunk 2]
```diff
@@ -98,3 +98,16 @@ The Sustainable SaaS (SusaaS) sample application has been built in a partner col
 Available for:
 
 [![Node.js logo](../assets/logos/nodejs.svg 'Link to the Sustainable SaaS (SusaaS) repository.'){style="height:2.5em; display:inline; margin:0 0.2em;"}](https://github.com/SAP-samples/btp-cap-multitenant-saas)
+
+
+## Partner Reference App {.github}
+
+With this application, we give SAP partners guidance on how to extend SAP ERP solutions with side-by-side cloud applications running on SAP Business Technology Platform (SAP BTP). It includes:
```
Comment (renejeglinsky) [inline get-started/samples.md:105]:
```suggestion
With this application, we give SAP partners guidance on how to extend SAP ERP solutions with side-by-side cloud applications running on SAP Business Technology Platform (SAP BTP). It includes the following:
```

**Review by renejeglinsky — APPROVED** [review]

**Comment by mrl-sap** [comment]
We are currently working on rephrasing the text, please do not merge yet.

**Comment by chgeo** [comment]
New text as suggested by @mrl-sap 

> The Partner Reference Application repository provides you with a “golden path” to becoming a SaaS provider of multi-tenant applications based on the SAP Business Technology Platform (SAP BTP).
> The guidance covers building, running, and integrating scalable full-stack cloud applications. It includes an ERP-agnostic design that lets you deliver your application as a side-by-side extension to consumers using any SAP solution, such as SAP S/4HANA Cloud, SAP Business One, and SAP Business ByDesign.
> 
> By using BTP services and the SAP Cloud Application Programming Model (CAP), your application meets SAP standards for enterprise-class business solutions. It offers a harmonized user experience and seamless integration, including:
> - centralized identity and access management,
> - a common launchpad,
> - cross-application front-end navigation,
> - and secure back-channel integration.
> 
> The repository includes the “Poetry Slam Manager” application as a ready-to-run example. It also provides tutorials on how to build the application from scratch using an incremental development approach.
> Based on this sample application, you will find the bill of materials and a sizing example. This addresses the question "Which BTP resources do I need to subscribe to and in what quantities?" and serves as a basis for cost calculation.
>

**Pull request merged**